Library/Homebrew/cask/audit.rb

```diff
@@ -47,44 +47,22 @@ def initialize(cask, appcast: nil, download: nil, quarantine: nil,
       @signing = signing
       @new_cask = new_cask
       @token_conflicts = token_conflicts
+      @only = only
+      @except = except
     end
 
     def run!
-      check_denylist
-      check_reverse_migration
-      check_required_stanzas
-      check_version
-      check_sha256
-      check_desc
-      check_url
-      check_unnecessary_verified
-      check_missing_verified
-      check_no_match
-      check_generic_artifacts
-      check_token_valid
-      check_token_bad_words
-      check_token_conflicts
-      check_languages
-      check_download
-      check_https_availability
-      check_single_pre_postflight
-      check_single_uninstall_zap
-      check_untrusted_pkg
-      livecheck_result = check_livecheck_version
-      check_hosting_with_livecheck(livecheck_result: livecheck_result)
-      check_appcast_and_livecheck
-      check_latest_with_appcast_or_livecheck
-      check_latest_with_auto_updates
-      check_stanza_requires_uninstall
-      check_appcast_contains_version
-      check_gitlab_repository
-      check_gitlab_repository_archived
-      check_gitlab_prerelease_version
-      check_github_repository
-      check_github_repository_archived
-      check_github_prerelease_version
-      check_bitbucket_repository
-      check_signing
+      only_audits = @only
+      except_audits = @except
```

Comment:
I think we should check somewhere if these are correct and actually exist. Currently we are only skipping if they match. If there's a typo it is ignored silently.

Reply:
We don't do that for formulae either. I'm fine adding it for both but that should be a new PR.